roles/identity-management/manage-rh-sso-identities/tasks/manage-users.yml

```diff
@@ -59,34 +76,32 @@
   when:
     - user.status == 201
 
-- name: "Search For Existing User"
-  uri:
-    url: "{{ rh_sso_protocol }}://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/admin/realms/{{ user_data.realm | mandatory }}/users?username={{ user_data.user_name }}"
-    method: GET
-    headers:
-      Authorization: "Bearer {{ rh_sso_token.json.access_token }}"
-  register: get_user
-  when:
-    - user.status == 409
+- name: "Update existing user: {{ user_data.user_name }}"
+  block:
+    - name: "Search For Existing User: {{ user_data.user_name }}"
+      uri:
+        url: "{{ rh_sso_protocol }}://{{ rh_sso_host }}:{{ rh_sso_port }}/auth/admin/realms/{{ user_data.realm | mandatory }}/users?username={{ user_data.user_name }}"
+        method: GET
+        headers:
+          Authorization: "Bearer {{ rh_sso_token.json.access_token }}"
+      register: get_user
```

Comment:
as these are registered facts, could we make it a bit more specific / unique than `get_user`  to avoid potential conflicts, overwrites, etc.? Perhaps `existing_rh_sso_user`?

Reply:
@oybed Went ahead and did some renaming from get_user -> get_sso_user (and some other associated facts). Let me know if this addresses the issue.